solution/text-extractor/backends/s3.py

```diff
@@ -29,10 +31,16 @@ def __init__(self, config: dict):
         except Exception as e:
             raise BackendInitException(f"failed to initialize AWS client: {str(e)}")
 
-    def get_file(self, uri: str) -> bytes:
+    def get_file(self, temp_directory: str, uri: str) -> bytes:
```

Comment:
The file is going to be stored in a temporary directory in the docker container as it is running.  When the lambda is done it deletes the file by itself.